hooks/notification/notification-templates/msteams-messageCard.njk

```diff
@@ -0,0 +1,66 @@
+type: message
```

Comment:
Is the `hooks/notification/notification-templates/.msteams-messageCard.njk.swp` binary really necessary? Looks a bit strange... Maybe add the file extension *.swp to .gitignore?

Reply:
Nope this was not necessary. Good catch :)